packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -300,102 +414,120 @@ const PollQuestion = ({event, classes}: {
  * If a postId is provided, we just give points to that post (ex. when on a post page).
  * Otherwise, we open a modal.
  */
```

Comment:
Can you update the comments in this file so that they are accurate? I don't think we have the modal anymore

Reply:
Yep, done